process/block/baseProcess_test.go

```diff
@@ -130,10 +131,16 @@ func createArgBaseProcessor(
 		BlockProcessingCutoffHandler:   &testscommon.BlockProcessingCutoffStub{},
 		ManagedPeersHolder:             &testscommon.ManagedPeersHolderStub{},
 		SentSignaturesTracker:          &testscommon.SentSignatureTrackerStub{},
-		RunTypeComponents:              components.GetRunTypeComponents(),
+		RunTypeComponents:              createRunTypeComponents(),
 	}
 }
 
+func createRunTypeComponents() runType.RunTypeComponentsHolder {
+	coreComp := components.GetCoreComponents(testscommon.GetGeneralConfig(), components.GetRunTypeCoreComponents())
```

Comment:
Same remark as everywhere else
I think everything was better before to just have a func to call which did everything.